src/components/Book.jsx

```diff
@@ -44,9 +44,9 @@ const Book = ({
       </div>
     </div>
     <div className="col d-flex justify-content-center align-items-start flex-column">
-      <p className="mb-0 text-uppercase completed">Current Chapter</p>
+      <p className="mb-0 text-uppercase completed current-chapter">Current Chapter</p>
       <p className="chapter">{chapter}</p>
-      <button type="button" className="btn btn-primary px-4">Update prograss</button>
+      <button type="button" className="btn btn-primary px-4 text-uppercase">Update prograss</button>
     </div>
   </div>
 );
```

Comment:
- [ ] Kindly check the color of the fonts from the template and fix them according to the previous reviewer.
![image](https://github.com/Gh-Reza/bookstore/assets/60083437/a11b4dba-c0c0-421e-a29e-27455966b7b5)